mint.json

```diff
@@ -264,7 +264,9 @@
                         "redis/sdks/ts/commands/json/del",
                         "redis/sdks/ts/commands/json/forget",
                         "redis/sdks/ts/commands/json/get",
+                        "redis/sdks/ts/commands/json/merge",
```

Comment:
Can you add these to this page too https://upstash-dx-1777.mintlify.app/redis/sdks/ts/commands/overview#json